packages/react/collapsible/src/Collapsible.test.tsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { axe } from 'jest-axe';
```

Comment:
Let me know if you think the tests that remain are ok to stay. I feel like these aren't covered so should stay.